packages/gatsby-recipes/README.md

```diff
@@ -241,7 +241,7 @@ DEBUG=true node --inspect-brk ./node_modules/.bin/gatsby recipes ./test.mdx
 
 Then, open up Chrome and click the node icon in dev tools.
```

Comment:
add info here about the `./recipe-server.log` with `DEBUG=true`?

Reply:
we've moved away from `recipe-server.log` as it's a lot easier to just run the API process in its own terminal window + it's tricky to make sure everything is logged when running things in the background.